apps/src/templates/GameButtons.jsx

```diff
@@ -13,9 +13,11 @@ import blankImg from '../../static/common_images/1x1.gif';
 
 const styles = {
   main: {
-    // common.scss provides an :after selector that ends up adding 18px of height
-    // to gameButtons. We want to get rid of that
-    marginBottom: -18
+    display: 'flex',
+    flexWrap: 'wrap',
+    columnGap: 9,
+    rowGap: 0,
```

Comment:
we might want to test how this looks in older browsers. looking at the [browser compatibility for column-gap](https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap#browser_compatibility), chrome 84 is when this became supported for flex elements. however, our UI test matrix tests chrome on Windows 7 Chrome 75:

https://github.com/code-dot-org/code-dot-org/blob/30da1c20b4cd4c8ceb137f0aea58051a4b93d213/dashboard/test/ui/browsers.json#L4-L11

there's a way to test this using saucelabs and a tunnel to your local environment, but first i want to see how concerned we are about this since our browser compatibility rules are changing in the new year. @aaronwaggener -- do you know how we should handle newer CSS properties like column-gap and row-gap? it seems fine that this isn't supported by IE11, but i'm not sure where our boundaries are for "kind of old but not that old version of chrome"

Reply:
I would be more concerned about iOS and macOS Safari. In the case of iOS, they first started supporting these properties this calendar year with v14.5. We haven't finalized a recommendation for which versions of Safari to test against (see [this comment](https://docs.google.com/document/d/17AIWp9Hzc2vwcVrH47aGU3kBLQbtiDCHfDvqrqEkE8A/edit?disco=AAAAQ0XcOnU) in the Browser Version Support Strategy doc). I think we will land on supporting at least one full year back to give time for school districts to update their fleet.

Given all of that, I recommend we hold on using these properties for any critical layout and ensure we/product are ok with how the degraded experience looks in other use cases.

Reply:
thanks aaron, that helps a lot! in that case, mike, i think we should move the `styles` we have here into common.scss (which is where all the other styles already are, so should actually be clearer in the end).

we need to do that because if we aren't using the column-gap and row-gap properties, we'll want to use properties that are better supported in older browsers, likely mixed with pseudoselectors (like last-child) and being aware of whether or not the document is in RTL mode. this means moving to SCSS because the `style` attribute is _very_ limited (and an anti-pattern) and doesn't support those features (it basically only supports base styles -- no nesting, pseudoselectors, media queries, etc)

Reply:
Agree with all the above from @madelynkasula. Also, just noting that it's great to see awareness and interest in using these newer properties. I am hopeful our browser support will catch up soon.

Reply:
In the meantime, we can use [`@supports` media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports) if we want to get the best of both worlds. We will end up with a little more CSS, but it will be obvious what direction we are trying to go and also make refactoring away rules for unsupported browsers easy in the future.
Any thoughts on this approach @madelynkasula? It's not a pattern I have seen us using. Especially for layout, I think it is worth considering.

Reply:
i think that's a great idea! i'd prefer that over avoiding new properties in cases like this, so great suggestion